packages/spartan/buildinfo.json

```diff
@@ -4,7 +4,7 @@
     "spartan": {
       "kind": "git",
       "git": "https://github.com/dcos/spartan.git",
-      "ref": "0b5243b8032bf475fb2c62ec4f92142d701f35ad",
+      "ref": "f7b2311c71fa833c182a13781af00a387632fb1b",
```

Comment:
@nlsun Please confirm that ref should be https://github.com/dcos/spartan/commit/e94d5661cd271387379fd41001b80a31a1b4eab8 from branch https://github.com/dcos/spartan/commits/dcos/1.9
I diffed the 1.9 branch against master against `nlsun/backport-blacklist`, and they've all got the same bits, even if different SHAs.